docker-compose/is/conf/identity-server/repository/conf/deployment.toml

```diff
@@ -9,11 +9,7 @@ password = "admin"
 create_admin_account = true
 
 [user_store]
-type = "read_write_ldap_unique_id"
-connection_url = "ldap://localhost:${Ports.EmbeddedLDAP.LDAPServerPort}"
-connection_name = "uid=admin,ou=system"
-connection_password = "admin"
-base_dn = "dc=wso2,dc=org"      #refers the base dn on which the user and group search bases will be generated
+type = "database_unique_id"
```

Comment:
why we have decided to change the default ldap to a jdbc usestore?

Reply:
we have removed the embedded ldap from IS6.0.0